src/components/ArticleList.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import SmallArticle from './SmallArticle';
+
+function ArticleList() {
+    const [isError, setIsError] = useState(false);
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
Great! Have a think about how we could use your error state to signal if there's been an error too